### ISSUE TITLE
Bug: initImageId always causes "Error: No job ID"

### ISSUE THREAD

**Issue by ejkkan** [Leonardo-Interactive/leonardo-ts-sdk#45]
testing to use an init image via the website, it works as expected. Trying to use the same init image id for your SDK makes the API throw errors

**Comment by anuraagnalluri** [comment]
Hi @ejkkan can you provide the specific error message along with the endpoint you're testing?

**Comment by ejkkan** [comment]
@anuraagnalluri 
I've created this example for most of the issues I've reported. [See here](https://github.com/ejkkan/leonardo-ai-template/tree/master) 
So please config it to make the setup relevant for each issue.
Notice that I'm using v1.28 specifically because of functionality being broken [as mentioned in](https://github.com/Leonardo-Interactive/leonardo-ts-sdk/issues/39)

**Comment by ejkkan** [comment]
I realize that I've phrased this issue poorly. "No job Id" is an internal error from my application. The underlying error is documented by comments and easily tested in the repo I provided

**Comment by speakeasybot** [comment]
Hi Erik, this should be addressed in SDK release `1.44.1` https://github.com/Leonardo-Interactive/leonardo-ts-sdk/releases/tag/v1.44.1

**Comment by ejkkan** [comment]
Still broken in `1.44.1`.
Neither imagePrompts array(which seems to be what Leonardo.ai website uses when doing generations) or initImageId is working.

You should see the faulty behavior if you update the SDK in the example repo I provided.

**Comment by alexadrake** [comment]
Hi @ejkkan , I used the script you provided  and did a fresh install of `1.44.1` and it worked as expected. One possible explanation is that the dependencies may not actually be updated, in which case deleting `node_modules` and doing a fresh install may resolve the issue.

Note that I had to fix the script to call `GenerateImages` instead of the inline script and I had to change the request fields to `camelCase`.
